### ISSUE TITLE
Loses ! comments in source files

### ISSUE THREAD

**Issue by turadg** [txs1992/stylus-converter#56]
Both Stylus and SCSS support a `/*!` comment to indicate to pass through compression, often used for copyright notices.
http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments
http://stylus-lang.com/docs/comments.html#multi-line-buffered

In version 0.4.1 the `!` is lost in conversion.

E..g,
```
/*!
 * http://stylus-lang.com/docs/comments.html#multi-line-buffered
 */
div {}
```
-->
```
/*
 * http://stylus-lang.com/docs/comments.html#multi-line-buffered
 */
div  {

}
```

**Comment by txs1992** [comment]
@turadg The problem has been located and will soon be fixed.

**Comment by txs1992** [comment]
Your question has been fixed in version 0.5.0.
